### DISCUSSION TITLE
Challenges in fine-tuning

### DISCUSSION THREAD

**Discussion by ep0p** [PaddlePaddle/PaddleOCR#14127] [Q&A]
I've been experimenting with fine-tuning paddleOCR custom dataset.
My objective is to improve recognition accuracy, especially on scanned documents that are noisy.

Here's my approach so far:

1. Created a custom dataset, applying noise (e.g., Gaussian noise, blur, salt and pepper) to around 60% of the images.
2. Fine-tuned, 50 epochs, batch_size (train and val):128

However, instead of improvement, I observed a drop in accuracy compared to the original models. I expected that fine-tuning would enhance the ocr performance, but this hasn’t been the case.

Could there be an issue with my approach, or am I potentially overlooking some key aspect of fine-tuning
